.github/workflows/test-performance-command.yml

```diff
@@ -0,0 +1,204 @@
+name: performance-test
```

Comment:
FYI, Greg confirmed on Slack that he wants to keep this one for now. I just reran/cancelled it so it will flag again in 3 months to revisit it

Reply:
I think I like this pattern.  We may need a whitelist at some point, but I actually think checking every 3 months we haven't run something is probably a good pattern to start with